aiocache/base.py

```diff
@@ -496,8 +509,9 @@ async def close(self, *args, _conn=None, **kwargs):
         logger.debug("CLOSE (%.4f)s", time.monotonic() - start)
         return ret
 
+    @abstractmethod
     async def _close(self, *args, **kwargs):
-        pass
+        raise NotImplementedError()
```

Comment:
Maybe this one shouldn't be abstract?

Many backends may not need to close, so a default `pass` is probably good. Whereas the other methods marked as abstract are actually needed to provide a fully functioning cache.